shell/utils/cypress.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
I have found a bunch of scripts already inside the shell (screenshot), so I thought it would be ok.
It should not take much to move into the root, so we may still run unit tests.

<img width="290" alt="Screenshot 2023-11-28 at 19 00 30" src="https://github.com/rancher/dashboard/assets/5009481/f8cc6047-2eb5-456d-b43e-a5cf8d065bec">

Reply:
Also please mind to avoid the import from shell as well at this point.

<img width="258" alt="Screenshot 2023-11-28 at 19 25 03" src="https://github.com/rancher/dashboard/assets/5009481/10414650-f3d9-4332-b969-ed0d9d98dc1b">